### ISSUE TITLE
Can't understand "collapsed" in index.html

### ISSUE THREAD

**Issue by apoorv-mishra** [BlackrockDigital/startbootstrap-creative#55]
Hi, 

I am new to open-source and this is the first project that I cloned. I just wanted to draw attention to line number 43 in index.html. It is as shown

```html
    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span> Menu <i class="fa fa-bars"></i>
                </button>
```
I have doubt with `class="navbar-toggle collapsed"`. What's the use of `collapsed` here? I searched, found about `collapse` but couldn't find anything related to `collapsed`. Am I missing something or is it just a typo?

**Comment by comhernandez28** [comment]
That is just the general collapsed class from bootstrap. Here is a link where you can read about it in more detail http://www.w3schools.com/Bootstrap/bootstrap_collapse.asp

**Comment by comhernandez28** [comment]
@apoorv-mishra That is just the general collapsed class from bootstrap. Here is a link where you can read about it in more detail http://www.w3schools.com/Bootstrap/bootstrap_collapse.asp

**Comment by apoorv-mishra** [comment]
Sorry, but I can only see `collapse` there, nothing like `collapsed`.

**Comment by comhernandez28** [comment]
@apoorv-mishra I see what you mean, you are talking about the class collapsed right? Well it appears to be a typo because I can not find any reference of it in the css. I'll check it out and do some testing then remove it and submit a pull request if it's not meant to be there.

**Comment by comhernandez28** [comment]
@apoorv-mishra I tested it out and its no typo as I thought. I cloned the entire repo just to realize that bit of code is directly copied from the bootstrap docs. What "collapsed" does is style the content that you expect to collapse. If you omit the class then the content you expect to collapse just will not display. It's probably hidden by something else and that content is redisplayed by the class. Maybe check the JS here if you're more interested https://github.com/twbs/bootstrap just do a Ctr-F for "collapsed"

**Comment by apoorv-mishra** [comment]
@comhernandez28  Yeah, I found out too. But when I omitted that class, I didn't find any problems. In fact, what I found while testing in dev tools (Chromium, Ubuntu 14.04) was that the class `collapsed` gets automatically added when I collapse the menu items by clicking on the "menu" button in top-right corner of the page ("menu" button appears as the browser window shrinks horizontally). Also it gets automatically removed when I do the opposite. So, I just wanted to ask if it is "redundant" to add `collapsed`? Because I feel that [collapse.js](https://github.com/twbs/bootstrap/blob/v4-dev/js/src/collapse.js) seems to handle it automatically. Although I'm not sure.
